app/src/main/java/io/github/droidkaigi/confsched2018/presentation/common/broadcastreceiver/NotificationBroadcastReceiver.kt

```diff
@@ -37,7 +38,7 @@ class NotificationBroadcastReceiver : BroadcastReceiver() {
         val notification = notificationBuilder(context, channelId)
                 .setContentTitle(title)
                 .setContentText(text)
-                .setContentIntent(pendingIntent)
+                .setFullScreenIntent(pendingIntent, true)
```

Comment:
I have a question 🙋 
Is setContentIntent no longer needed?

Reply:
I think so.
Because setFullScreenIntent is also requires PendingIntent and kicked it when user taps notification (or simply kicked PendingIntent without showing heads-up notification when the device not support heads-up notification).

Reply:
Thank you for confirming it 👍

Reply:
Sorry, If I ignore the heads-up notification, Can I tap notification in status bar? 📊

Reply:
Sorry, I had not thought such case...
I don't actually know what "ignore" means, but if it means "Pull down the heads-up notification" I say "Yes, you can", means "Swipe horizontally the heads-up notification" I say "No, you can't".

Reply:
Thank you for description 👍